pkg/cluster/cluster.go

```diff
@@ -16,25 +16,30 @@ import (
 )
 
 type Cluster struct {
-	logger        micrologger.Logger
-	runner        runner.Runner
-	fs            afero.Fs
-	remoteCluster bool
+	logger          micrologger.Logger
+	runner          runner.Runner
+	fs              afero.Fs
+	existingCluster bool
+	remoteCluster   bool
 }
 
-func New(logger micrologger.Logger, fs afero.Fs, runner runner.Runner, remoteCluster bool) *Cluster {
+func New(logger micrologger.Logger, fs afero.Fs, runner runner.Runner, existingCluster bool, remoteCluster bool) *Cluster {
 	return &Cluster{
-		logger:        logger,
-		runner:        runner,
-		fs:            fs,
-		remoteCluster: remoteCluster,
+		logger:          logger,
+		runner:          runner,
+		fs:              fs,
+		existingCluster: existingCluster,
+		remoteCluster:   remoteCluster,
 	}
 }
 
 // Create is a Task that creates a remote cluster or, if we
 // are using a local one, puts in place the required files for
 // later access to it
 func (c *Cluster) Create() error {
+	if c.existingCluster {
+		return nil
```

Comment:
maybe it make sense to create kubeconfig here instead of in kvm-operator e2e tests?

Reply:
:+1: As discussed on slack this would involve more changes, can be addressed in separate PR(s) if needed.